release/models/aft/openconfig-aft-common.yang

```diff
@@ -244,11 +258,156 @@ submodule openconfig-aft-common {
           }
         }
 
+        container encap-headers {
+          description
+            "Container for packet encapsulation headers.  When present, this
+            container indicates that encapsulation of the packet matching the
+            next-hop is performed using a stack of one or more packets defined
+            in the list encap-header.
+
+            Each entry in the list must indicate an encapsulation type and
+            populate a container with the parameters for that encapsulation
+            header.";
+
+          list encap-header {
+            description
+              "A list of headers added on top of a packet.  The first entry
```

Comment:
Can we say what the "first entry" means here -- it seems like it is is the lowest index.

Must the indices start at 0?

Reply:
done